stories/theme/index.stories.tsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import {
+  Title,
+  Description,
+  Heading,
+  Source,
+  Stories,
+} from '@storybook/addon-docs/blocks';
+import type { Meta } from '@storybook/react';
+import { BREAKPOINTS } from 'src/constants';
+
+export * from './boxShadows.stories';
+export * from './colors.stories';
+export * from './fonts.stories';
+export * from './typography.stories';
+
+const styledComponentSource =
+  // eslint-disable-next-line no-template-curly-in-string
+  'export const StyledComponent = styled.div`\n  color: ${({ theme }) => theme.COLORS.primary};\n`';
+
+const cssPropSource =
+  // eslint-disable-next-line no-template-curly-in-string
+  '<Component css={(theme: ThemeType) => css`color: ${theme.COLORS.primary}`}>';
```

Comment:
I think we can remove the in-line typing to reduce confusion. I don't think our other code usage examples include types that aren't present in the source.

Reply:
our code does not include the ThemeType => that's true. I discovered I could type it after I did the Colors refactor. I will remove it from here yes I agree. I guess I added so people know they can use the typing otherwise the theme will have `any` type and they can make typos (I know I will use it in some cases)